include/seastar/net/tls.hh

```diff
@@ -305,6 +305,12 @@ namespace tls {
         sstring _priority;
     };
 
+    struct tls_options {
+        // Optionally indicate to the client if it should wait for an EOF from the server
+        // after sending the BYE message. By default, the wait is hardcoded to 10 seconds.
+        bool wait_for_eof_on_shutdown = true;
```

Comment:
Please use doxygen syntax (and verify it renders correctly locally).

Reply:
Done.